tests/test_comparison_template_lib.py

```diff
@@ -0,0 +1,149 @@
+import pandas as pd
+import pytest
+
+import splink.duckdb.duckdb_comparison_template_library as ctld
+import splink.spark.spark_comparison_template_library as ctls
+from splink.duckdb.duckdb_linker import DuckDBLinker
+from splink.spark.spark_linker import SparkLinker
+
+
+def test_duckdb_date_comparison_run():
+    ctld.date_comparison("date")
+
+
+def test_spark_date_comparison_run():
+    ctls.date_comparison("date")
```

Comment:
So, I was thinking you could do something like so:

```suggestion
@pytest.mark.parametrize(
    ("cll"),
    [
        pytest.param(ctld, id="DuckDB date_comparison Integration Test"),
        pytest.param(ctls, id="Spark date_comparison Integration Test"),
    ],
)
def test_duckdb_date_comparison_run(cll):
    cll.date_comparison("date")
```

Reply:
Cool, that's what I have put in now